### PR TITLE
[PBNTR-65] Center Align Text for Status Dialogs

### PR DESCRIPTION
**Pull request by gavhuang**
**What does this PR do?**
- [Runway](https://nitro.powerhrg.com/runway/backlog_items/PBNTR-65)
- Centers the text for status dialogs by adding a class name: [`dialog_status_text_align`](https://github.com/powerhome/playbook/compare/PBNTR-65-Dialog-Title-Body?expand=1#diff-e0614f355c6addc50e729055d662c40e0486bf3b34ee83d2578a9cff2790ffa4R185)

**Screenshots:**
| Before | After |
|--------|--------|
| Screenshot 2023-07-20 at 12 38 20 PM | Screenshot 2023-07-20 at 12 30 31 PM |

**How to test?** Steps to confirm the desired behavior:
1. Go to the Dialog kit
2. Open a status Dialog variant
3. Put in some long text - it should be centered

#### Checklist:
- [X] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [X] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss [hunk 1]
```diff
@@ -258,26 +262,30 @@
...
-	}	
+	}
 }
 
-//styles specific to rails version of kit
+// styles specific to rails version of kit
 // rails version has own wrapper because of the way the overlay functions for the HTML dialog used to create this
 .pb_dialog_wrapper_rails {
-	$medium: 500px;
+		$medium: 500px;
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss:273]: indentation seems a little off here! But code looks good!

**Review by Israel-Molestina — APPROVED** [review]

**Review by nidaqg — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb [hunk 2]
```diff
@@ -5,38 +5,38 @@
...
-                <%= pb_rails("dialog/dialog_header", props: {title: object.title, id: object.id }) %>
+                <%= pb_rails("dialog/dialog_header", props: { title: object.title, id: object.id }) %>
             <% end %>
             <% if object.status === "" && object.text %>
                 <%= pb_rails("dialog/dialog_body", props: { text: object.text }) %>
             <% end %>
             <% if object.status != "" %>
-                <%= pb_rails("dialog/dialog_body", props: {padding: "md"}) do %>
-                    <%= pb_rails("flex", props: {align: "center", orientation: "column"}) do %>
+                <%= pb_rails("dialog/dialog_body", props: { classname: "dialog_status_text_align" ,padding: "md" }) do %>
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb:14]: Why are we only adding this for this specific use case (this one conditional)? 
Does it not make sense to add when the user also passes in text?
 
I reached out to a designer just to confirm that we are indeed wanting to change this completely. We normally would want to cut an alpha for something like this that changes default styling on a kit. (we are changing the way it looks on people's actual UI in nitro/other clients).

Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb:14]: Why are we only adding this for this specific use case (this one conditional)? 
Does it not make sense to add when the user also passes in text?
 
I reached out to a designer just to confirm that we are indeed wanting to change this completely. We normally would want to cut an alpha for something like this that changes default styling on a kit. (we are changing the way it looks on people's actual UI in nitro/other clients).

Comment (gavhuang) [reply playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb:14]: @jasperfurniss Colin mentioned that it was a requested change in [this archived story - PLAY-859](https://nitro.powerhrg.com/runway/backlog_items/PLAY-859). 

My apologies if I misread the story. I can cut an alpha if this is a desired change.

**Review by jasperfurniss — APPROVED** [review]

File: playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb (hunk 2 above)
Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_dialog/dialog.html.erb:14]: Nope! My fault! @gavhuang I looked into it, and I misread our own code!

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**